docs/source/install/deb.rst

```diff
@@ -30,13 +30,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 Setup repositories
 ~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+The following script will detect your platform and architecture and setup the repo accordingly. It'll also install the GPG key for repo signing.
```

Comment:
`st2bootstrap-deb.sh` - I don't like the idea of wget raw version of the script. I remember we avoided this in packaging to allow users doing everything manually with thinking and understanding.
So with formatted version I think people read few lines of that script first before blind wget/bash.

But you feel free to commit directly to `feature/packagecloud_latest` branch in this `st2docs` repo.